python/dgl/distributed/kvstore.py

```diff
@@ -0,0 +1,1008 @@
+"""Define distributed kvstore"""
+
+import os
+import time
+import random
+import numpy as np
+
+from . import rpc
+from .graph_partition_book import PartitionPolicy
+
+from .. import backend as F
+from .._ffi.ndarray import empty_shared_mem
+
+def check_file_exists(filename):
+    """Check if file exists.
+
+    Parameters
+    ----------
+    filename : str
+        Path of file
+
+    Returns
+    -------
+    True if file exists.
+    """
+    if os.path.exists(filename):
+        return True
+    return False
```

Comment:
this can be simply `os.path.exists(filename)`. i'm not sure if it's necessary to add a function for this.